doc/examples/test-examples.js

```diff
@@ -0,0 +1,36 @@
+const { readdirSync, statSync } = require('fs');
+const { join } = require('path');
+const { spawn } = require('child_process');
+const exampleDirs = readdirSync(__dirname)
+	.map(dir => join(__dirname, dir))
+	.filter(dir => statSync(dir).isDirectory());
+
+async function runner(dir) {
```

Comment:
Please remove the `async` keyword here, since `await` is not used in this function.

Reply:
👍